core/src/main/java/io/questdb/griffin/engine/functions/catalogue/TableListFunctionFactory.java

```diff
@@ -243,7 +243,7 @@ public boolean open(CharSequence tableName) {
                         // Pre-read as much as possible to skip record instead of failing on column fetch
                         tableId = tableReaderMetadata.getTableId();
                         maxUncommittedRows = tableReaderMetadata.getMaxUncommittedRows();
-                        commitLag = tableReaderMetadata.getCommitLag();
+                        commitLag = tableReaderMetadata.getO3MaxLag();
```

Comment:
Variable name does not match the right side name.

Reply:
Please rename the table column tool

Reply:
renamed in https://github.com/questdb/questdb/pull/2759/commits/be85cbfb3d59b70be8180622869ea92a58033841